include/rocksdb/utilities/transaction.h

```diff
@@ -316,6 +316,14 @@ class Transaction {
                               const SliceParts& key) = 0;
   virtual Status SingleDelete(const SliceParts& key) = 0;
 
+  virtual Status DeleteRange(const ReadOptions& read_options,
```

Comment:
We should include the warning comment above the `DeleteRange` declaration in "include/rocksdb/db.h" about the negative performance impact.

Reply:
@ajkr I already updated it.